apps/kubernetes-express-otel/express-otel-app/instrumentation.js

```diff
@@ -0,0 +1,17 @@
+//instrumentation.js from 
+//    https://opentelemetry.io/docs/languages/js/getting-started/nodejs/*/
+//    https://opentelemetry.io/docs/languages/js/exporters/
+const { NodeSDK } = require('@opentelemetry/sdk-node');
+const { getNodeAutoInstrumentations } = require('@opentelemetry/auto-instrumentations-node');
+const { OTLPTraceExporter } = require('@opentelemetry/exporter-trace-otlp-proto');
+const { diag, DiagConsoleLogger, DiagLogLevel } = require('@opentelemetry/api');
+
+// Enable debug mode for the SDK
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG);
+
+const sdk = new NodeSDK({
+  traceExporter: new OTLPTraceExporter({url: process.env.OTEL_EXPORTER_OTLP_ENDPOINT}),
```

Comment:
After following the previous suggestion the config here is no longer needed:
```suggestion
  traceExporter: new OTLPTraceExporter(),
```
It was needed before because if not it would try and append the traces specific path and end up trying to send things to `http://$(HOST_IP):4318/v1/traces/v1/traces`